OpenQA/Isotovideo/Runner.pm

```diff
@@ -49,6 +62,7 @@ sub run ($self) {
         $ch->check_asserted_screen if defined($ch->tags);
     }
     $ch->stop_command_processing;
+    $self->_exit_code_from_test_results();
```

Comment:
Do we always want to compute the exit code? If yes, why not return it here?

Otherwise I would have expected that this is called if needed like before

Reply:
i am looking at the second option.

Reply:
yes, I also think this is the better option